docs/src/intro.md

```diff
@@ -9,6 +9,13 @@ hide:
   <p align="center">Create a full Earth Observation API with Metadata, Raster and Vector services.</p>
 </p>
 
+
+## 🌍 eoAPI: An Open-Source Community Project
```

Comment:
It's weird to talk about the community before people even know what the tools is.